src/main/java/frc/robot/maps/FrancoisMap.java

```diff
@@ -123,8 +123,8 @@ public PIDSparkMax shooterWheel() {
     public ControlPanelMap getControlPanelMap() {
         return new ControlPanelMap() {
             @Override
-            public SendableSpeedController spinner() {
-                SendableSpeedController motor = SendableSpeedController.wrap(controlPanel);
+            public SmartSpeedController spinner() {
+                SmartSpeedController motor = SmartSpeedController.wrap(controlPanel);
```

Comment:
Line seems a bit redundant with the below line, just merge the two like what happens elsewhere

Reply:
doesn't the control panel speed controller need to be defined up at the top of FrancoisMap so that we can use it the control panel gyro?

Reply:
It is defined at the top of the robot map class. This is a wrapper class that we don't use for anything here and can be removed.